packages/playground-examples/copy/es/JavaScript/Functions with JavaScript/Generic Functions.ts

```diff
@@ -0,0 +1,103 @@
+//// { title: 'Funciones genéricas' }
+
+// La genericidad proporciona una forma de utilizar tipos
+// como variables en otros tipos. Meta.
```

Comment:
> Los genéricos proporcionan una forma de utilizar tipos como variables en otros tipos.

Also, what is Meta on the original file? Maybe some extra word or was on purpose?

Reply:
Sí, está en el original. Se refiere a la primera oración de que la genericidad permite usar tipos como variables en otros tipos (algo así como metatipos). No estoy seguro cuál sería la forma más idiomática de traducirlo.

Reply:
Ah, utiliza `Metadatos` para traducir ese `Meta` :)

Reply:
:thinking: No creo que metadatos sea exactamente la traducción más feliz. En el texto original parece más bien una forma de guiño o broma al lector.